Assets/MRTK/SDK/Features/UX/Scripts/Buttons/ButtonIconSet.cs

```diff
@@ -244,6 +244,16 @@ public bool EditorDrawCharIconSelector(uint currentChar, out bool foundChar, out
                 }
             }
 
+            if (currentSelection >= 0)
+            {
+                EditorGUILayout.BeginHorizontal();
+
+                EditorGUILayout.LabelField("Current Icon Name", EditorStyles.boldLabel);
+                EditorGUILayout.LabelField(quadIcons[currentSelection].name);
```

Comment:
Does `quadIcons` need to change for these three? Seems like this one might need to be
```suggestion
                EditorGUILayout.LabelField(charIcons[currentSelection].name);
```